### DISCUSSION TITLE
Shulkers drop 2 shells

### DISCUSSION THREAD

**Discussion by srnyx** [gnembon/fabric-carpet#784] [Ideas]
Shulker boxes will always drop 2 shells when killed

**Comment by James103** [comment]
Do you mean that there should be a feature to make it so Shulkers always drop 2 shells? If so, then [Vanilla Tweaks](https://vanillatweaks.net/picker/datapacks/) already includes such a feature.

**Reply by srnyx** [reply]
Ok, I guess I'll just keeping using that
